petab/visualize/helper_functions.py

```diff
@@ -468,7 +468,8 @@ def check_ex_exp_columns(
         raise NotImplementedError(
             "Column \'simulationConditionId\' is missing in measurement "
             "file. ")
-    if MEASUREMENT not in exp_data.columns:
+    if MEASUREMENT not in exp_data.columns and SIMULATION not in \
```

Comment:
this only allows the case if measurement & simulation is not available, is this intended? Can this happen? Isnt it either measurement is not available or simulation?

Reply:
and for the measurement data, 'simulation' is not needed?

Reply:
a dataframe has to have eiter `simulation` or `meaurement. This is a cheap way to make the function validate measument dfs and simulation dfs at the same time

Reply:
I will give the check_ex_exp_col function an option for simulation to make it more clean